### PR TITLE
Remove usage of torch.onnx symbolic_registry

### LINKED ISSUE #13008: [Training] pytorch exporter deleted symbolic_registry  while ort still uses it
### Describe the issue

this pytorch PR https://github.com/pytorch/pytorch/pull/84382 removed "torch.onnx.symbolic_registry", while ort still uses [it ](https://github.com/microsoft/onnxruntime/blob/14365b67a0de0acc1e6423477d7363bd5a98f0d7/onnxruntime/python/tools/pytorch_export_contrib_ops.py#L19)

@justinchuby could you give some suggestion on what kind of change ort need to take according to this torch change?

### To reproduce

the python code `import torch.onnx.symbolic_registry as sym_registry` will fail to run.

### Urgency

_No response_

### ONNX Runtime Installation

Released Package

### ONNX Runtime Version or Commit ID

ort nightly

### PyTorch Version

torch nightly

### Execution Provider

CUDA

### Execution Provider Library Version

_No response_

### PR DESCRIPTION
**Pull request by justinchuby**
**Description**: symbolic_registry is deprecated in torch.onnx. This PR removes its usage. 

Fixes #13008

### CODE REVIEW

File: onnxruntime/python/tools/pytorch_export_contrib_ops.py [hunk 1]
```diff
@@ -98,8 +98,9 @@ def unregister():
         try:
             torch.onnx.unregister_custom_op_symbolic(name, _OPSET_VERSION)
         except AttributeError:
+            from torch.onnx import symbolic_registry
```
Comment (wschin) [inline onnxruntime/python/tools/pytorch_export_contrib_ops.py:101]: Just out of curiosity, why do you import `symbolic_registry` here? It was imported right below `from torch.onnx import symbolic_helper`.

**Review by wschin — DISMISSED** [review]

**Comment by pranavsharma** [comment]
fyi: The fix for the Linux eager model pipeline failure is here https://github.com/microsoft/onnxruntime/pull/13000.

File: onnxruntime/python/tools/pytorch_export_contrib_ops.py (hunk 1 above)
Comment (justinchuby) [reply onnxruntime/python/tools/pytorch_export_contrib_ops.py:101]: The symbolic_registry module was removed in the latest pytorch; we are importing here for backwards compatibility. Added a comment.

**Review by zhijxu-MS — APPROVED** [review]

**Comment by pranavsharma** [comment]
You need to bring in this commit for the training pipeline to pass https://github.com/microsoft/onnxruntime/commit/8ea742b50721e8c8f781d2e531f963344286b0f4

**Pull request merged**